node/public/JS/bmi-calc.js

```diff
@@ -0,0 +1,8 @@
+function BMI() {
+    var h = document.getElementById('h').value;
```

Comment:
don't use variables like w and h, use height and weight

Reply:
okay, made the changes